tests/Doctrine/Tests/Inflector/Rules/Dutch/DutchFunctionalTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+use Doctrine\Inflector\Language;
+use Doctrine\Tests\Inflector\Rules\LanguageFunctionalTest;
+
+class DutchFunctionalTest extends LanguageFunctionalTest
+{
+    /**
+     * @return list<array{string, string}>
+     */
+    public function dataSampleWords(): array
+    {
+        return [
+            ['schip', 'schepen'],
+            ['stad', 'steden'],
+            ['gelid', 'gelederen'],
+            // @todo more
+            ['weerman', 'weermannen'],
+            ['ei', 'eieren'],
+            ['rij', 'rijen'],
+            ['mogelijkheid', 'mogelijkheden'],
+            ['adres', 'adressen'],
+            ['olie', 'oliën'],
+            ['industrie', 'industrieën'],
+            ['lid', 'leden'],
+            ['smid', 'smeden'],
+            ['kalf', 'kalveren'],
+            ['lam', 'lammeren'],
+            ['koe', 'koeien'],
+            ['vlo', 'vlooien'],
+            ['leerrede', 'leerredenen'],
+            ['lende', 'lendenen'],
+            ['genius', 'geniën'],
+            ['aanbod', 'aanbiedingen'],
+            ['dank', 'dankbetuigingen'],
+            ['gedrag', 'gedragingen'],
+            ['genot', 'genietingen'],
+            ['lof', 'lofbetuigingen'],
+            ['qaestrices', 'quaestrix'],
+            ['matrices', 'matrix'],
```

Comment:
Singular and plural are exchanged here
```suggestion
            ['matrix', 'matrices'],
```